Gopkg.toml

```diff
@@ -36,7 +36,8 @@ required = [
 # Our master branch tracks knative/pkg master or a release.
 [[override]]
   name = "knative.dev/pkg"
-  branch = "master"
+  branch = "add_isunknown"
+  source = "github.com/capri-xiyue/pkg"
```

Comment:
I'm waiting for the approval of https://github.com/knative/pkg/pull/965. Once it gets merged, I will change the Gopkg.toml back to master branch

Reply:
I would recommend not allowing knative/pkg#965 in, it will break the integrations for all other knative resources and there is no reason for the new methods in the manager.

Reply:
Could you share me with the example that  knative/pkg#965 will break the integrations for all other knative resources?